### DISCUSSION TITLE
Alt Nerve WADs

### DISCUSSION THREAD

**Discussion by exmensa** [Owlet7/wadfusion#34] [Q&A] [answered]
Hello!  Really appreciate your picking up where wadsmoosh left off, I love it :)

I'm curious if I'm doing something wrong or if it's just not implemented - if I understand correctly, putting an original WAD + a Unity or KEX WAD in source_wads will use the data from the original and just the updated widescreen resources from Unity/KEX, is that not an option for Nerve as well?  I see mention of the graphic for KEX in the source but the program doesn't seem to recognize NerveUnity.wad or NerveKEX.wad alongside the usual Nerve.wad the way it does with the others.

**Comment by Owlet7** [comment, marked as answer]
NRFTL is treated differently than the Unity/KEX IWADs. WF will recognise if the newer versions of `nerve.wad` are used, and extract their intermission screen without any additional work from the user.

WadSmoosh originally required making an exact copy of the WAD and renaming it to `nerveu.wad` before it would extract the asset, but this isn't the case anymore.

**Reply by exmensa** [reply]
I think I was just reading too far into 'These versions can also be used as the main WADs for extraction, but do keep in mind that they are censored'.  I take it there's no other unwanted modifications in the newer NRFTLs and you can just rename one to use as Nerve.wad without missing out on anything?  If that's the case, nothing to worry about.

**Reply by Owlet7** [reply]
Yup, that's the case. I updated the readme to make that less confusing.
